Doc/library/types.rst

```diff
@@ -143,6 +143,12 @@ Standard names are defined for the following types:
    Note that the audited arguments may not match the names or positions
    required by the initializer.
 
+   .. method:: CodeType.replace(**kwargs)
+
+     Return a new code object with new specified fields.
```

Comment:
Suggestion: I would word it as some variation of:

"Return a copy of the code object with new values for the specified fields" 

(Basically being explicit that it will be copied and that the fields provided will be modified in the copy)

Reply:
sounds great! should I update the docstring as well (I copied this text from there)?

Reply:
> sounds great! should I update the docstring as well (I copied this text from there)?

Yep, let's keep them consistent :)

Reply:
🎉 and done!  thanks for the help

Reply:
Thank you for the PR 😉